data/maps/DewfordTown_PokemonCenter_2F/scripts.inc

```diff
@@ -1,19 +1,20 @@
 DewfordTown_PokemonCenter_2F_MapScripts:: @ 81FC615
-	map_script MAP_SCRIPT_ON_FRAME_TABLE, DewfordTown_PokemonCenter_2F_MapScript2_276C3B
-	map_script MAP_SCRIPT_ON_WARP_INTO_MAP_TABLE, DewfordTown_PokemonCenter_2F_MapScript2_276B6C
-	map_script MAP_SCRIPT_ON_LOAD, DewfordTown_PokemonCenter_2F_MapScript1_276BBE
-	map_script MAP_SCRIPT_ON_TRANSITION, PokemonCenter_2F_OnTransition
+	map_script MAP_SCRIPT_ON_FRAME_TABLE, CableClub_MapScript2_OnFrame
```

Comment:
`_MapScript2` `_MapScript1`, and `_MapScript` are redundant information, which should be removed from map script labels.  For example, `CableClub_OnFrameTable` or `CableClub_OnTransition`.  (This is how the rest of the codebase is doing it, too)